testframework/src/main/java/net/neoforged/testframework/impl/test/MethodBasedGameTestTest.java

```diff
@@ -16,9 +17,11 @@
 public class MethodBasedGameTestTest extends AbstractTest.Dynamic {
     protected MethodHandle handle;
     private final Method method;
+    private final Class<? extends GameTestHelper> helperType;
 
-    public MethodBasedGameTestTest(Method method) {
+    public MethodBasedGameTestTest(Method method, Class<?> helperType) {
```

Comment:
You should cast the class where the constructor is used, rather than have the constructor accept a wildcard.

Reply:
Done.